cmd/snap/cmd_debug_validate_seed_test.go

```diff
@@ -0,0 +1,68 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main_test
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	. "gopkg.in/check.v1"
+
+	snap "github.com/snapcore/snapd/cmd/snap"
+)
+
+func (s *SnapSuite) TestDebugValidateSeedHappy(c *C) {
+	tmpf := filepath.Join(c.MkDir(), "seed.yaml")
+	err := ioutil.WriteFile(tmpf, []byte(`
+snaps:
+ -
+   name: core
+   channel: foo
+   file: bar
+ -
+   name: gnome-foo
+   channel: foo
+   name: xxx
```

Comment:
duplicate `name`

Reply:
Thanks, fix this (and the other one) with more realistic inputs.